pkgs/data/misc/geolite-legacy/default.nix

```diff
@@ -2,17 +2,17 @@
 
 stdenv.mkDerivation rec {
   pname = "geolite-legacy";
-  version = "20230901";
+  version = "20240720";
 
   # We use Arch Linux package as a snapshot, because upstream database is updated in-place.
   geoip = fetchurl {
```

Comment:
If you use `fetchzip` instead the tar commands from build should become unnecessary